raspyrfm_client/device/manufacturer/universal/HX2262Compatible.py

```diff
@@ -9,37 +9,51 @@ class HX2262Compatible(Device):
     _seqHi = [(_hi, _lo), (_hi, _lo)]
     _seqFl = [(_lo, _hi), (_hi, _lo)]
 
-    _bits = []
+    _connair_params = {'repetitions': 5, 'pauselen': 5600, 'steplen': 350}
 
     from raspyrfm_client.device.manufacturer import manufacturer_constants
     def __init__(self, manufacturer: str = manufacturer_constants.UNIVERSAL, model: str = manufacturer_constants.HX2262):
-        super().__init__(manufacturer, model, {'repetitions': 5, 'pauselen': 5600, 'steplen': 375})
-
-    def set_channel_config(self, **channel_arguments) -> None:
-        """
-        :param channel_arguments: bits=['0'|'1'|'f']
-        """
-        for dip in self._bits:
-            if dip not in channel_arguments:
-                raise ValueError("arguments should contain key \"" + str(dip) + "\"")
-            self._bits.append(channel_arguments[dip])
-
-        self._channel = channel_arguments
+        super().__init__(manufacturer, model)
 
     def get_supported_actions(self) -> [str]:
         return [actions.ON, actions.OFF]
         
-    def set_bits(self, bits: []):
-        self._bits = bits
-
-    def generate_code(self, action: str = None) -> str:
-        tuples = []
+    def get_bits(self, action: str):
+        """
+        This method should be implemented by inheriting classes
+        :return: bit array, 12 bits á '0'|'1'|'f'
+        """
+        raise NotImplementedError
+        
+    def calc_int_bits(self, value, num_bits):
+        bits = []
+        for i in range(num_bits):
+            bits.append(self._h if (value & 1<<i) != 0 else self._l)
+        return bits
+        
+    def calc_match_bits(self, value, num_bits):
+        bits = []
+        for i in range(num_bits):
+            bits.append(self._h if value == i else self._l)
+        return bits
+        
+    def calc_dip_bits(self, dips):
```

Comment:
Please add documentation to all methods, even if they are only used for internal use.